packages/hub/src/server/ai/analytics/index.ts

```diff
@@ -0,0 +1,132 @@
+import "server-only";
+
+import * as Prisma from "@prisma/client";
+
+import { CodeArtifact, Workflow } from "@quri/squiggle-ai/server";
+
+import { prisma } from "@/prisma";
+import { getAiCodec } from "@/server/ai/utils";
+import { v2WorkflowDataSchema } from "@/server/ai/v2_0";
+import { checkRootUser } from "@/server/helpers";
+
+async function loadWorkflows() {
+  await checkRootUser();
```

Comment:
Note on this and other files in `src/server/`:

I have a long-term plan on how to gradually migrate from GraphQL to RSC by building up abstractions for the data layer.

I could document it more, but basically:
- it follows the standard best practices from Next.js docs
- there's a lot of overlap between `src/server/` and `src/graphql/helpers/*`, I hope to reorganize it later (out of scope for this PR)